discovery_engine/lib/src/discovery_engine_base.dart

```diff
@@ -87,7 +87,7 @@ class DiscoveryEngine {
       entryPoint ??= kIsWeb ? null : entry_point.main;
       final manager = await DiscoveryEngineManager.create(entryPoint);
       final initEvent = ClientEvent.init(configuration);
-      final response = await manager.send(initEvent);
+      final response = await manager.send(initEvent, shouldTimeout: false);
```

Comment:
hmm... the time needed to fetch the assets is related to the number of assets and to connection speed, so not sure if we can set for a send method a specific timeout during which we expect a response..

maybe we should decouple responding to the `Init` event from the init process.. but not sure if this would be so straightforward to implement in the PR. Maybe as a follow up

would you be ok, with just disabling the timeout for the init event, until we have a better solution? (I can add a new ticket to cover it if you want)

Reply:
yes sure